bigtable/tests/system.py

```diff
@@ -137,8 +140,13 @@ def test_reload(self):
 
         instance.reload()
         self.assertEqual(instance.display_name, Config.INSTANCE.display_name)
+        self.assertEqual(instance.labels, Config.INSTANCE.labels)
+        self.assertEqual(instance.instance_type, _PRODUCTION)
+
+    def test_create_instance_defaults(self):
+        from google.cloud.bigtable_admin_v2 import enums
+        _PRODUCTION = enums.Instance.Type.PRODUCTION
 
-    def test_create_instance(self):
         ALT_INSTANCE_ID = 'new' + unique_resource_id('-')
```

Comment:
In order to deal with the possibility that `Instance.delete` may have latency, it would be better to use a different prefix here (e.g., `'new-defaults'`) than in the `test_create_instance` testcase below.

Reply:
Changed prefix to `'new-def'` to avoid creating `Cluster Id` longer than 30 chars.